modules/containers-signature-verify-enable.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-container-signature.adoc
+
+[id="containers-signature-verify-enable_{context}"]
+= Enabling signature verification for Red Hat Container Registries
+Enabling container signature validation requires files that link the registry URLs to the sigstore and then specifies the keys which verify the images.
+
+.Procedure
+. Create the files which link the registry URLs to the sigstore and specifies the key to verify the image.
+.. policy.json
++
+[source,terminal]
+----
+cat > policy.json <<EOF
+{
+  "default": [
+    {
+      "type": "insecureAcceptAnything"
+    }
+  ],
+  "transports": {
+    "docker": {
+      "registry.access.redhat.com": [
+        {
+          "type": "signedBy",
+          "keyType": "GPGKeys",
+          "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
+        }
+      ],
+      "registry.redhat.io": [
+        {
+          "type": "signedBy",
+          "keyType": "GPGKeys",
+          "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
+        }
+      ]
+    },
+    "docker-daemon": {
+      "": [
+        {
+          "type": "insecureAcceptAnything"
+        }
+      ]
+    }
+  }
+}
+EOF
+----
+
+.. registry.access.redhat.com.yaml
++
+[source,terminal]
+----
+cat <<EOF > registry.access.redhat.com.yaml
+docker:
+     registry.access.redhat.com:
+         sigstore: https://access.redhat.com/webassets/docker/content/sigstore
+EOF
+----
+
+.. registry.redhat.io.yaml
++
+[source,terminal]
+----
+cat <<EOF > registry.redhat.io.yaml
+docker:
+     registry.redhat.io:
+         sigstore: https://registry.redhat.io/containers/sigstore
+EOF
+----
+
+. Next, set the files with a `base64` encode that will be used for the machine config template.
```

Comment:
Don't need the _Next,_. Being an ordered step conveys that by itself.

Reply:
Is _encode_ a noun? On-line dictionaries don't suggest it is. Maybe it is in practice. We don't seem to use it as a noun. 
I see _base64-encoded_ a lot. _Generate the base64-encoded user name and password_

Reply:
Revised as:

"Set the files with a `base64` encode format that will be used for the machine config template."